testsuite/testsuite-jwt/src/test/resources/keys/jwt-test.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQD0gYeBVyaJOfiaBsj8DLKmQyZMU30KrxA4kg2nvBEOHyVRSwNKaocrSAwF44VTC8Sz0yqWP6p7TbWMCKvzeuBJxf8pE1r5qcgsDdbXR3aPv7JubEvrJ0mKnFkm0M7CGkRGOf28gUy7rHURcTSl+bUUbkYcWN3mBVfijrVQR4uOHMfN6RfpdgVIp9Y5rF/6/RFG1aBMVexTU9ZWblEMhdfQqN5GNvZ7c7YYS9w4K1TKs/CRygoKegsvI2i3DFRifmFwXCFvKALNCuja00bnqV4UKTEuOQh52uWQ3vmisI91M5OjBo7A95nesANmjGCT581NN9mgi3Oj0DQs9xWIgGKD hbraun@MacBook-Pro.fritz.box
```

Comment:
It doesn't matter in this test, but it might be misleading. This isn't a PEM-formatted public key. Feel free to use e.g. the `*.pem` files from https://github.com/Ladicek/swarm-test-suite/tree/master/microprofile/microprofile-jwt-1.0/src/test/resources

Reply:
+1

Reply:
Hi Ladislav, should truststores be supported too ? That may be a diff issue though; the users may want to just to point to the store containing the imported public key

Reply:
Agree, but that would be a different JIRA. Care to file one? :-)